README.md

```diff
@@ -14,12 +14,56 @@ You can install the required python libraries by executing `./install_packages.s
 
 ## Usage
 
-1. To upload plugins to ChRIS: `./sshUploadPlugins.sh`
+1. Upload plugins to ChRIS. There are 2 options for doing this:
```

Comment:
Really great, detailed and clear instructions here. Helps improve documentation for COVID-Net!